Kudu.Contracts/Jobs/FunctionEnvelope.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Kudu.Contracts.Infrastructure;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+
+namespace Kudu.Contracts.Jobs
+{
+    public class FunctionEnvelope : INamedObject
```

Comment:
Unless "Envelope" is a naming that is already established for other types, you might consider "FunctionDescriptor" instead.